addons/hr_homeworking/models/hr_employee.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+from odoo.addons.hr_homeworking.models.hr_homeworking import DAYS, DAYS_STRING
+
+from collections import defaultdict
+from datetime import timedelta
+from dateutil.rrule import rrule, WEEKLY
+
+
+class HrEmployeeBase(models.AbstractModel):
+    _inherit = "hr.employee.base"
+
+    monday_location_id = fields.Many2one('hr.work.location', string='Monday')
+    tuesday_location_id = fields.Many2one('hr.work.location', string='Tuesday')
+    wednesday_location_id = fields.Many2one('hr.work.location', string='Wednesday')
+    thursday_location_id = fields.Many2one('hr.work.location', string='Thursday')
+    friday_location_id = fields.Many2one('hr.work.location', string='Friday')
+    saturday_location_id = fields.Many2one('hr.work.location', string='Saturday')
+    sunday_location_id = fields.Many2one('hr.work.location', string='Sunday')
+    hr_icon_display = fields.Selection(selection_add=[('presence_home', 'At Home'),
+                                                      ('presence_office', 'At Office'),
+                                                      ('presence_other', 'At Other')])
+
+    @api.depends(lambda _: DAYS)
+    def _compute_presence_icon(self):
+        super()._compute_presence_icon()
+        today = DAYS_STRING[fields.Date.today().weekday()]
+        for employee in self:
+            today_employee_location_id = getattr(employee, f'{today}_location_id', False)
```

Comment:
I don't understand this comment, line 31 gets the worklocation for that day, I can get rid of the default value since an empty recordset will be falsy ( when a user didnt configure a worklocation for this day ) but I dont understand the part about mappings